src/components/comments.jsx

```diff
@@ -1,20 +1,57 @@
-import { getCommentsByReview } from "../api";
+import { getCommentsByReview, postComment } from "../api";
 import "./review.css";
 import { useParams } from "react-router-dom";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 
-function Comments({ comments, setComments, loading, setLoading }) {
+function Comments({ comments, setComments }) {
   const { review_id } = useParams();
+  const [newComment, setNewComment] = useState("");
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    postComment(newComment, review_id).then(() => {
+      getCommentsByReview(review_id).then((data) => {
+        setComments(data);
+        setLoadingComments(false);
+      });
+    });
+    setNewComment("");
+  };
 
   useEffect(() => {
     getCommentsByReview(review_id).then((data) => {
       setComments(data);
-      setLoading(false);
+      setLoadingComments(false);
     });
   }, [review_id]);
 
+  if (loadingComments)
+    return (
+      <div id="reloading" className="reloading">
+        Loading
+      </div>
+    );
+
   return (
     <ul className="comment-list">
+      <div className="comment-section">
+        <img
+          className="pfp"
+          src="https://scontent.fman4-2.fna.fbcdn.net/v/t1.18169-9/1044725_10151527841570922_2110461867_n.jpg?_nc_cat=108&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=qzcXY7CFmlcAX9SVbW9&_nc_ht=scontent.fman4-2.fna&oh=00_AfDNLFtojnACZzG2HJZ47hgd88pBRCoEIMZysUuWNBs2Cw&oe=63B7EA0B"
+        ></img>
+        <form onSubmit={handleSubmit}>
+          <input
```

Comment:
This is a single line input box. I'd look at using textarea to allow multi-line comments 👍

Reply:
added text area and removed the ability to resize it. Textbox will scroll down if users write too much. may chance that with css later